--- mod_gearman-1.0.10.orig/worker/initscript.in	2011-09-05 00:40:02.000000000 +0100
+++ mod_gearman-1.0.10/worker/initscript.in	2011-09-05 00:38:44.000000000 +0100
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# chkconfig: 2345 80 30
+# description: Mod-gearman worker daemon.
+# processname: mod_gearman_worker
+# config: /etc/gearmand.conf
+
 ### BEGIN INIT INFO
 # Provides:          mod_gearman_worker
 # Required-Start:    $local_fs
