diff -r -u mod_gearman-0.6.orig/worker/initscript.in mod_gearman-0.6/worker/initscript.in
--- mod_gearman-0.6.orig/worker/initscript.in	2010-10-09 00:39:57.000000000 +0100
+++ mod_gearman-0.6/worker/initscript.in	2010-10-24 02:49:53.000000000 +0100
@@ -1,5 +1,10 @@
 #!/bin/sh
 
+# chkconfig: 2345 80 30
+# description: Mod-gearman worker daemon.
+# processname: mod_gearman_worker
+# config: /etc/gearmand.conf
+
 ### BEGIN INIT INFO
 # Provides:          mod_gearman_worker
 # Required-Start:    $local_fs
@@ -14,7 +19,7 @@
 
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 DAEMON=%WORKERBIN%
-NAME=mod_gm_worker
+NAME=mod_gearman_worker
 CONFIG=%CONFIG%
 PIDFILE=%PIDFILE%
 USER=%USER%
